openshift_cli_installer/libs/user_input.py

```diff
@@ -169,6 +169,24 @@ def is_platform_supported(self):
                 self.logger.error("\n".join(missing_platforms))
                 raise click.Abort()
 
+    def is_installer_log_level_supported(self):
+        supported_log_levels = ["debug", "info", "warn", "error"]
+        aws_ipi_clusters = [
+            _cluster["platform"]
+            for _cluster in self.clusters
+            if _cluster["platform"] == AWS_STR
+        ]
+        if aws_ipi_clusters:
+            for _cluster in aws_ipi_clusters:
+                log_level = _cluster.get("log-level", "error")
+                if log_level not in supported_log_levels:
+                    self.logger.error(
+                        f"Log Level {log_level} for cluster {_cluster['name']}"
```

Comment:
We want to report on all clusters, collect the bad cluster and log them all if any

Reply:
Done, added all unsupported log level and clusters in error msg